### ISSUE TITLE
Patcher removal?

### ISSUE THREAD

**Issue by smoi** [brett8883/DJI_Super-Patcher#154]
There's no mention on how to remove the patch or if it's safe to update to the latest firmware.
Is it enough to update to the latest firmware to remove the patch?

**Comment by lovrenco** [comment]
I would like to know this as well. What is the procedure to flash back factory firmware (without the patch) in case of need?

**Comment by Weirdoz** [comment]
Just flash the actual firmware and the patch is gone.

**Comment by Weirdoz** [comment]
> There's no mention on how to remove the patch or if it's safe to update to the latest firmware.
> Is it enough to update to the latest firmware to remove the patch?

Yes just flash a "normal" firmware from dji. Or the one you flashed before applying the patch, just flash without patching.
